src/main/kotlin/com/csbroker/apiserver/model/ShortProblem.kt

```diff
@@ -84,7 +84,17 @@ class ShortProblem(
             scoreList.count { it == this.score },
             scoreList.count { it != this.score },
             totalSolved,
-            this.answer.length
+            this.answer.length,
+            this.isEnglish()
         )
     }
+
+    private fun isEnglish(): Boolean {
+        for (c in this.answer) {
```

Comment:
혹시 여기 this.answer에 입력될 때 공백이 포함되는지에 대해 validation 하는 코드가 따로 있나요?
있다면 이대로 가도 좋고 따로 없이 약속만 한 상태라면 혹시 모를 공백은 제거 후에 확인하는게 좋을 것 같아요!
```suggestion
        for (c in this.answer.replace("\\s".toRegex(), "")) {
```